controller/controller.go

```diff
@@ -119,6 +119,8 @@ type Controller struct {
 	nextRunAtMux sync.Mutex
 	// DNS record types that will be considered for management
 	ManagedRecordTypes []string
+	// MinInterval is used as window for batching events
+	MinInterval time.Duration
```

Comment:
Can we use a more descriptive variable? For example: 

```suggestion
	MinEventSyncInterval time.Duration
```